src/main/java/mate/academy/controllers/driver/LoginController.java

```diff
@@ -0,0 +1,41 @@
+package mate.academy.controllers.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.lib.Injector;
+import mate.academy.model.Driver;
+import mate.academy.security.AuthenticationService;
+
+public class LoginController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private final AuthenticationService authenticationService =
+            (AuthenticationService) injector.getInstance(AuthenticationService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/drivers/login.jsp").forward(req, resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/login.jsp").forward(req, resp);
```
login page should be some sort of a root page